### ISSUE TITLE
train dataset is shuffled regardless of seed

### ISSUE THREAD

**Issue by goonbamm** [ArrowLuo/CLIP4Clip#108]
Thank you for your great code.

Actually, I found that the dataset sampler does not get any seed, so it use default value '0' as seed. It seems that the seed only affects the initialization of the model.

So I think we need add the following parameters for all samplers.

~~~python
train_sampler = torch.utils.data.distributed.DistributedSampler(msvd_dataset)
~~~

~~~python
# default: shuffle = True, seed = 0
train_sampler = torch.utils.data.distributed.DistributedSampler(msvd_dataset, shuffle=True, seed=args.seed)
~~~

I apologize in advance if I'm wrong.
